packages/widget/rollup.config.js

```diff
@@ -7,7 +7,7 @@ import packageJson from './package.json';
 export default [
   {
     input: ['./src/index.ts'],
-    external: ['react', 'react-dom'],
```

Comment:
I don't think you need to make `@r2wc/react-to-web-component` in external packages